### PR TITLE
[MM-28140] e2e/system_console/customization: add T1026

### PR DESCRIPTION
**Pull request by mgdelacroix**
#### Summary
Adds T1026: Custom Branding - Name character limit

#### Ticket Link
https://mattermost.atlassian.net/browse/MM-28140

### CODE REVIEW

File: e2e/cypress/integration/system_console/ui_and_api/customization_spec.js [hunk 1]
```diff
@@ -396,6 +396,28 @@ describe('Customization', () => {
             expect(config.TeamSettings.EnableCustomBrand).to.equal(false);
         });
     });
+
+    it('T1026 - Custom Branding - Name character limit', () => {
```
Comment (saturninoabril) [inline e2e/cypress/integration/system_console/ui_and_api/customization_spec.js:400]:
```suggestion
    it('MM-T1026 - Custom Branding - Name character limit', () => {
```

**Review by saturninoabril — CHANGES_REQUESTED** [review]
Thanks @mgdelacroix! LGTM, same comment for folder and test ID.

**Review by wiggin77 — APPROVED** [review]
Test looks good to me.  I'm uneasy about just truncating the input field for sitename at 30 chars instead of error.

**Review by saturninoabril — APPROVED** [review]
Thanks @mgdelacroix! LGTM, tested and passed.

@wiggin77 Actual behavior is not truncating but not allowing to input more characters when exceeding the limit. But I agree to have an error shown rather than silently disallowing more inputs. Maybe bring up to UX for further consideration?

**Review by lindy65 — APPROVED** [review]
Thanks @mgdelacroix - looks good to me 👍 

@wiggin77 @saturninoabril - I've created https://mattermost.atlassian.net/browse/MM-28528 for UX to look at the issue of max characters in the Site Name and Site Description fields.

**Pull request merged**